app/controllers/v0/sessions_controller.rb

```diff
@@ -66,6 +66,24 @@ def saml_callback
       stats(:total)
     end
 
+    def saml_callback_v2
```

Comment:
initially just added this but given the amount of logic related to saml in here, we probably just wanna create a duplicate sessions controller too?

Reply:
Yes, let's actually create a sessions_controller under `app/controllers/v1/sessions_controller`